src/experimental.ts

```diff
@@ -1,3 +1,4 @@
 export * from "./experimental/job";
 export * from "./experimental/golem_network";
 export * from "./experimental/reputation";
+export * from "./experimental/gvmi";
```

Comment:
I'm curious why you want to have this as an experimental feature? Are there any known problems that prevent it from being an actual feature?

Reply:
Hmm now that I think about it there is no good reason, it can be moved to the root export